thingworx-import/Entities.xml

```diff
@@ -678,11 +678,162 @@ TyqZNc0vshhVwE/OK4OsFVTXATO/uu5FZ5fgfXfotCBIJBTXfJuCCwGuOahZo/N+tl7BN2PormEP
+let m2WebResultInfoTable = me.ReadAllTagsLiveGenericService({
+    ewonName: ewonName /* STRING */,
+    useCached: useCached /* BOOLEAN */
+});
+
+let m2WebResultInfoTableRowMatch = m2WebResultInfoTable.Find({
+    TagName: "\""+tagName+"\""
+});
+
+let result = null;
+try {
+    // Get value and cast to int (closest equivalent to unsigned int)
+    result = parseInt(m2WebResultInfoTableRowMatch.Value);
+} catch (err) {
+    logger.warn("A value could not be found on Ewon " + ewonName + " for the tag requested " + tagName);
+}]]></code></Row></Rows></ConfigurationTable></ConfigurationTables></ServiceImplementation><ServiceImplementation description="" handlerName="Script" name="ReadFloatTagLiveGenericService"><ConfigurationTables><ConfigurationTable dataShapeName="" description="" isMultiRow="false" name="Script" ordinal="0"><DataShape><FieldDefinitions><FieldDefinition baseType="STRING" description="code" name="code" ordinal="0"/></FieldDefinitions></DataShape><Rows><Row><code><![CDATA[// Read live tag values for Ewon
+let m2WebResultInfoTable = me.ReadAllTagsLiveGenericService({
+    ewonName: ewonName /* STRING */,
+    useCached: useCached /* BOOLEAN */
+});
+
+let m2WebResultInfoTableRowMatch = m2WebResultInfoTable.Find({
+    TagName: "\""+tagName+"\""
+});
+
+let result = null;
+try {
+    // Get value and cast to float
+    result = parseFloat(m2WebResultInfoTableRowMatch.Value);
+} catch (err) {
+    logger.warn("A value could not be found on Ewon " + ewonName + " for the tag requested " + tagName);
+}]]></code></Row></Rows></ConfigurationTable></ConfigurationTables></ServiceImplementation><ServiceImplementation description="" handlerName="Script" name="ReadIntegerTagLiveGenericService"><ConfigurationTables><ConfigurationTable dataShapeName="" description="" isMultiRow="false" name="Script" ordinal="0"><DataShape><FieldDefinitions><FieldDefinition baseType="STRING" description="code" name="code" ordinal="0"/></FieldDefinitions></DataShape><Rows><Row><code><![CDATA[// Read live tag values for Ewon
+let m2WebResultInfoTable = me.ReadAllTagsLiveGenericService({
+    ewonName: ewonName /* STRING */,
+    useCached: useCached /* BOOLEAN */
+});
+
+let m2WebResultInfoTableRowMatch = m2WebResultInfoTable.Find({
+    TagName: "\""+tagName+"\""
+});
+
+let result = null;
+try {
+    // Get value and cast to int
+    result = parseInt(m2WebResultInfoTableRowMatch.Value);
+} catch (err) {
+    logger.warn("A value could not be found on Ewon " + ewonName + " for the tag requested " + tagName);
```

Comment:
You might want some separator around the tagName and ewon name. 
Ex:
_A value could not be found on Ewon HoustonLine1 for the tag requested BeltSpeed._
Looks a bit weird.

A value could not be found on Ewon "HoustonLine1" for the tag requested "BeltSpeed"
or
A value could not be found on Ewon (HoustonLine1) for the tag requested (BeltSpeed)
might be an improvement